lte/gateway/c/core/oai/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ project(MagmaCore)
 
 include(CMakeOptionsMacros.txt)
 include($ENV{MAGMA_ROOT}/orc8r/gateway/c/common/CMakeProtoMacros.txt)
+include($ENV{MAGMA_ROOT}/lte/idl/oai/experimental/CMakeFlatMacros.txt)
```

Comment:
shouldn't this be `..lte/oai/experimental`? what is the purpose for the `idl` folder outside oai module?

Reply:
I did it like this because of lte/protos/oai/mme_nas_state.proto etc.
I choose a more generic name for the folder
I can do "lte/oai/experimental"

Reply:
let's go with `lte/oai/experimental`